deploy-and-test-cf.yml

```diff
@@ -1,4 +1,4 @@
-resources:
+in_parallelresources:
```

Comment:
@MarcSchunk does `in_parallelresources` really exist, or is that a typo? I could not find it in the Concourse docs.

Reply:
@suhlig Marc already made another commit which fixes it. Have a look at the latest commit.

Reply:
not really sure how I managed to do this. I was surprised myself. Apologies.

As @petergtz said, its already fixed, just pull master.